packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -28,19 +44,55 @@ interface SquadPageHeaderProps {
 const MAX_WIDTH = 'laptopL:max-w-[38.5rem]';
 const Divider = classed('span', 'flex flex-1 h-px bg-border-subtlest-tertiary');
 
+interface SquadStatProps {
+  count: number;
+  label: string;
+}
+
+const SquadStat = ({ count, label }: SquadStatProps) => (
+  <span className="flex flex-row text-text-tertiary typo-footnote">
+    <strong className="mr-1 text-text-primary typo-subhead">
+      {largeNumberFormat(count)}
+    </strong>
+    {label}
+  </span>
+);
+
 export function SquadPageHeader({
```

Comment:
This file is massive, think we could have split it into some components.

Reply:
Let's maybe create a ticket to revisit this. We've done quite some testing already and don't want to change too much with the components as we'll be forced to run through it again.